packages/react-badge/src/components/CounterBadge/CounterBadge.types.ts

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+import { ComponentProps, ObjectShorthandProps } from '@fluentui/react-utilities';
+import { BadgeProps } from '../Badge/index';
+
+/**
+ * {@docCategory CounterBadge}
+ */
+export type CounterBadgeAppearance = 'filled' | 'ghost';
```

Comment:
I don't see these used anywhere, do you need explicit types for these ?

If consumers really wanted to reuse types, they could do smth like `CounterBadgeProps['shape']`